File: CANDY123.java
import java.util.*;
import java.lang.*;
import java.io.*;

class CANDY123
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        while(t>0)
        {
            t--;
            int a = sc.nextInt();
            int b = sc.nextInt();
            for(int i=1;i<=1000;i++)
            {
                // odd - Limak turn 
                if(i%2 == 1) {
                    if(a>=i)
                        a = a-i;
                    else {
                        System.out.println("Bob");
                        break;
                    }
                }
                // even - Bob turn
                else {
                    if(b>=i)
                        b = b - i;
                    else {
                        System.out.println("Limak");
                        break;
                    }
                }
            }
        }
        sc.close();
    }
}

File: CANDY123.py
t = int(input())
while t>0:
    t-=1
    a, b = input().split()
    # print("a = {} and b = {}".format(a, b))
    a = int(a)
    b = int(b)
    for i in range(1, 1001):
        if i%2 == 0:
            # even case - Bob turn
            if b>=i:
                b -= i
            else:
                print("Limak")
                break
        else:
            # odd case - Limak turn
            if a>=i:
                a -= i
            else:
                print("Bob")
                break